### ISSUE TITLE
Can Schema Guru handle nested structures like div in HTML and g in SVG (if they were converted to JSON)

### ISSUE THREAD

**Issue by coderextreme** [snowplow/schema-guru#118]
Here's a sample JSON file http://coderextreme.net/SeaStarGroup.json [here's the license for this file: http://x3dgraphics.com/examples/X3dForWebAuthors/license.html ].  I will download schema-guru try this unless you get to me soon.

**Comment by jbeemster** [comment]
Hey @coderextreme you can try out schema-guru here (http://schemaguru.snowplowanalytics.com/).  As for the JSON you supplied it does appear to have successfully produced a JsonSchema.

**Comment by coderextreme** [comment]
Yeah, I just tried it.  It produced a JSON schema, but if you look, Transforms are specified over and over.  We just want one Transform object/attribute in the schema I think, if possible.

**Comment by coderextreme** [comment]
If one could specify a JSON schema to start derived from an XML Schema, like one from Jsonix Schema Compiler or JAXB/Jsonix, or conversion from OWL, I think that would work better.  In this case, our attribute names, arrays and comments have a prefix, @, - or #, so I don't think that would work without some coding.  Or if I could take 2 schemas with similar names and copy the names from the schema with the prefix to the schema without the prefix, that would be wonderful.

**Comment by coderextreme** [comment]
Ugh, too many transforms.

John

**Comment by chuwy** [comment]
Hi @coderextreme! Sorry for late reply, I missied this issue somehow.
Unfortunately (if I understand everything correct) that is not possible now to derive these nested objects as separate subschemas. Although it would be a nice feature to have, at the same time it's hard to implement and not corresponds with our current needs.
I'll schedule it as far-out milestone, but I couldn't promise it will be implemented.

**Comment by coderextreme** [comment]
I'm pretty sure it could be implemented with a map or dynamic object.
